plugins/SLURM-Integration/ufm_slurm.conf

```diff
@@ -19,4 +19,6 @@ sharp_allocation=false
 # By setting partially_alloc to false, UFM SHARP allocation API will fail the SHArP allocation request if at least one node does not exist in the fabric.
 partially_alloc=true
 # Name of UFM-SLURM integration logging file
-log_file_name=ufm_slurm.log
\ No newline at end of file
+log_file_name=ufm_slurm.log
+# By setting this parameter to a non-zero value, the running Slurm job will terminate if either ufm-prolog.sh or ufm-epilog.sh exits with a non-zero status.
```

Comment:
please add:
by default, the value is set to 0, which means, allowing Slurm job to continue running even if the ufm-prolog.sh or ufm-epilog.sh exit with non-zero status.